app/src/lib/lookups.js

```diff
@@ -107,7 +104,7 @@ async function check_ontology(lookup_token) {
 	if (lookup_token.form_results.length) {
 		const forms = lookup_token.form_results
 		lookup_token.lookup_results = lookup_token.lookup_results
-			.filter(lookup => forms.some(form => form.part_of_speech === lookup.part_of_speech))
+			.filter(lookup => forms.some(form => form.form === 'stem' || form.part_of_speech === lookup.part_of_speech))
```

Comment:
Fixes for a word like 'correct' where the form lookup just results in the Adjective, but the ontology lookup results in both the Adjective and Verb. Since 'correct' is an uninflected form, it cannot be considered exclusive to the one part of speech.